.github/bin/release-plan/discord-announce.mjs

```diff
@@ -0,0 +1,46 @@
+const defaultPayload = {
+	content: '',
+	embeds: [
+		{
+			title: '',
+			url: '',
+			description: '',
+			color: 3963146,
+		}
+	]
+};
+const BASE_URL = 'https://github.com/csstools/postcss-plugins/tree/main';
+const getChangelog = (changelog) => {
+	const firstUnreleasedIndex = changelog.indexOf('### Unreleased');
+	const firstLineBreakIndex = changelog.indexOf('\n', firstUnreleasedIndex);
+	const secondRelease = changelog.indexOf('###', firstLineBreakIndex);
```

Comment:
We might need to do a pass over all changelogs to make sure they all use `###` and not `##`

Reply:
Already happened 🎉